common/ownership_voucher/ownership_voucher_test.go

```diff
@@ -0,0 +1,80 @@
+package ownershipvoucher
```

Comment:
ditto: license header and package comment

Reply:
Done.